### PR TITLE
Fix deleting DBs in test for PSQL and document setup

### PR DESCRIPTION
**Pull request by EnriqueL8**
Small PR to document that a Postgres instance needs to be running for the e2e of the go-sdk and make sure that the name of the table is not lowercased if not the tables will not get cleaned up after the test

### CODE REVIEW

File: go-sdk/integration-test/e2e_test.go [hunk 1]
```diff
@@ -657,8 +657,9 @@ func TestPostgresStorage(t *testing.T) {
 	assert.NoError(t, err)
 
 	defer func() {
-		db.Exec("DROP TABLE " + core.TreeRootsTable)
-		db.Exec("DROP TABLE " + core.NodesTablePrefix + "test_1")
+		// Table name needs to be wrapped in quotes if not it will be lowercased
+		db.Exec("DROP TABLE " + "\"" + core.TreeRootsTable + "\"")
+		db.Exec("DROP TABLE " + "\"" + core.NodesTablePrefix + "test_1\"")
```
Comment (Chengxuan) [inline go-sdk/integration-test/e2e_test.go:662]: I think the code should check the exec commands returns no errors to avoid this happening again

Comment (EnriqueL8) [reply go-sdk/integration-test/e2e_test.go:662]: good idea

File: go-sdk/integration-test/e2e_test.go [hunk 2]
```diff
@@ -592,113 +588,3 @@ func testConcurrentInsertion(t *testing.T, alice *babyjub.PublicKey, values []in
...
-func TestKeyManager(t *testing.T) {
-	keypair := decryptKeyStorev3(t)
-
-	keyEntry := key.NewKeyEntryFromPrivateKeyBytes([32]byte(keypair.PrivateKeyBytes()))
-	assert.NotNil(t, keyEntry)
-
-	assert.NotNil(t, keyEntry.PrivateKey)
-	assert.NotNil(t, keyEntry.PublicKey)
-	assert.NotNil(t, keyEntry.PrivateKeyForZkp)
-}
```
Comment (Chengxuan) [inline go-sdk/integration-test/e2e_test.go:704]: was this an intended removal?

**Review by jimthematrix — APPROVED** [review]
lgtm except for the question from @Chengxuan

**Comment by jimthematrix** [comment]
sorry missed this in the review, you said:

> make sure that the name of the table is not lowercased if not the tables will not get cleaned up after the test

where was this addressed?

File: go-sdk/integration-test/e2e_test.go (hunk 2 above)
Comment (EnriqueL8) [reply go-sdk/integration-test/e2e_test.go:704]: oh no my bad

**Comment by EnriqueL8** [comment]
Here https://github.com/hyperledger-labs/zeto/pull/40/files#diff-7f04ceab68afe0e2e8989c32363b39b3f55907daf03d7bc4e7ebc312b328274cR101-R104 @jimthematrix

**Review by Chengxuan — APPROVED** [review]

**Pull request merged**